Fix linux dev (#7)
awehook/blink-mind-desktop: README.md

# BlinkMindDesktop

BlinkMindDesktop is an open source mind-map and outliner app.

> BlinkMindDesktop is an MIT licensed open source project, and the latest version will always be downloadable for free from the GitHub release page. BlinkMindDesktop is still in development. 

Reviewer: Another nit:

```suggestion
BlinkMindDesktop is an MIT-licensed open source project, and the latest version will always be downloadable for free from the GitHub release page. BlinkMindDesktop is still in development. 
```

BlinkMindDesktop is an MIT licensed open source project, and the latest version will always be downloadable for free from the GitHub release page. BlinkMindDesktop is still in development. 

> BlinkMindDesktop use the open source mind map library [BlinkMind](https://github.com/awehook/blink-mind) for mindmap rendering.

Reviewer: While we're at it:

```suggestion
BlinkMindDesktop uses the open source mind map library [BlinkMind](https://github.com/awehook/blink-mind) for mind map rendering.
```

| Cloud Storage  | planing |  Save and open file from clound storage(icloud/onedrive/google drive).  |

## For Dev

### Develop

To develop locally, you need to have Chrome installed with the [Immutable.js Object Formatter extension](https://chrome.google.com/webstore/detail/immutablejs-object-format/hgldghadipiblonfkkicmgcbbijnpeog).

> First run download the code and the relevant submodules:

Reviewer: ```suggestion
First download the code and the relevant submodules:
```
